app/actions/websocket/index.ts

```diff
@@ -474,11 +475,8 @@ function handleEvent(msg: WebSocketMessage) {
 }
 
 function handleHelloEvent(msg: WebSocketMessage) {
-    const serverVersion = msg.data.server_version;
-    if (serverVersion && Client4.serverVersion !== serverVersion) {
-        Client4.serverVersion = serverVersion;
-        EventEmitter.emit(General.SERVER_VERSION_CHANGED, serverVersion);
-    }
+    const serverVersion = semverFromServerVersion(msg.data.server_version);
+    EventEmitter.emit(General.SERVER_VERSION_CHANGED, serverVersion);
```

Comment:
Shouldn't we check the server version stored to make sure it changed before emitting the event?

Reply:
We could although the check happens on the event listener